portal/templates/flask_user/_macros.html

```diff
@@ -116,3 +116,28 @@
     <a href="{%-if config.MOVEMER_LINK_URL-%}{{config.MOVEMER_LINK_URL}}{%-endif-%}" class="logo-link" title="{{ _('Movember') }}" target="_blank">
     {%-if white_theme -%}<img id="footerLockup" src="{{PORTAL}}{{url_for('static',filename='img/Movember-Footer-Logo.png')}}" alt="{{_('Movember Logo')}}" />{%-else-%}<img src="{{PORTAL}}{{url_for('static',filename='img/movember_logo.jpg')}}" alt="Movember Logo"/>{%-endif-%}</a>
 {%- endmacro %}
+{% macro checkCookieMonster(target="") -%}
```

Comment:
I think this might be more appropriate in a different directory as it's not specific to flask-user...